src/Raven.Server/Documents/Queries/Parser/QueryParser.cs

```diff
@@ -2387,7 +2391,8 @@ internal bool Field(out FieldExpression token)
                 {
                     if (Scanner.String(out var str, fieldPath: part > 1))
                     {
-                        if (part == 1)
+                        if (part == 1 || 
+                            part == 2 && _insideTimeSeriesBody && parts[0].Value == _fromAlias)
```

Comment:
>1 here, no?

Reply:
2, we do `part++` before : https://github.com/ravendb/ravendb/blob/7e2149e5329607bf04c6dc5da9d8c51a1e084b1e/src/Raven.Server/Documents/Queries/Parser/QueryParser.cs#L2390